.gitignore

```diff
@@ -49,3 +49,18 @@ doc/_build
 .coverage
 cover
 .ipynb_checkpoints
+
+
+# virtual environment #
+###################
+venv/
```

Comment:
Why do we need to add venv here?

Reply:
When activating a virtual environment,  ex : using pycharm (IDE), it will create a subfolder : emva1288/venv , where the interpreter (venv/bin) and  and dependancies (venv/lib) from the requirements.txt will be installed. We wouldn't want to include these and other artifacts in this folder from polluting the repo. Therefore its added to exclusions list.

Reply:
The venv is outside (and  independent) of a python project. This is not required here in general no one creates a venv inside a python project.

Reply:
fair enough, I'll update the .gitignore